psi4/src/core.cc

```diff
@@ -979,13 +996,36 @@ bool psi4_python_module_initialize() {
     static char* argv = (char*)"";
     for_rtl_init_(&argc, &argv);
 #endif
+    
+#ifdef USING_BrianQC
+    const char* brianEnableEnv = getenv("BRIANQC_ENABLE");
```

Comment:
is environment variables how the user customarily activates BrianQC? In psi, `cmake -DENABLE_<external>=ON` is a signal to detect-or-build-or-die-trying, and runtime activation would normally be through an option (read_options.cc). Pretty much no envvar control. But if BrianQC users with other programs are accustomed to envvar control, there's something to be said for keeping that consistency.

Reply:
I will have to talk this over with the other developers. For the sake of consistency between host programs, we would probably like to keep the env var, but we can create an option as well. One advantage of the env var over an input file option is that is allows the user (and us) to use the same input verbatim with and without Brian, for the sake of comparing performace, precision, etc.

Reply:
I expect having both envvar and option will work (though we might try to capture envvar and route the instruction through an option). QC programs are already awful at exposing their state, and I worry about envvar leaving no trace about what was running, except maybe a line in text output. Agree about utility for testing.